tests/equilibria/test_equilibrium.py

```diff
@@ -329,7 +329,7 @@ def test_betapliip_profile(self, shape):
             self.B_0,
             shape=shape,
             li_min_iter=0,
-            li_rel_tol=rel_tol,
+            li_reltol=rel_tol,
```

Comment:
oh this should be changed back
```suggestion
            li_rel_tol=rel_tol,
```